docs/verify/simulate-with-qiskit-aer.ipynb

```diff
@@ -0,0 +1,259 @@
+{
```

Comment:
<p>can you extract the code comment and put it as an admonition component outside the code block? It's quite lengthy so I think better to move it out the code block. Also should the TODO be removed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/302/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
I don't think Jupyter supports admonition components so I used a paragraph quote.

I added the TODO. Do you think it should be removed? If so we need to remember to update the notebook after Qiskit Aer 0.14. If the notebook is tested in CI that is not an issue because the notebook will simply break.

Reply:
Ok, I opened https://github.com/Qiskit/documentation/issues/319 and I'll remove the TODO.